libbeat/processors/add_docker_metadata/config.go

```diff
@@ -33,6 +33,7 @@ type Config struct {
 	SourceIndex  int               `config:"match_source_index"` // Index in the source path split by / to look for container ID.
 	MatchPIDs    []string          `config:"match_pids"`         // A list of fields containing process IDs (PIDs).
 	HostFS       string            `config:"system.hostfs"`      // Specifies the mount point of the host’s filesystem for use in monitoring a host from within a container.
+	DeDot        bool              `config:"labels.dedot"`       // If set to true, replace dots in labels with `_`.
```

Comment:
The new option should be added to documentation too.

Reply:
Hey @jsoriano, I only find  one spot in libbeat/_meta/config.reference.yml to update the documentation for DoDot here. Am I missing other important spot?

Reply:
awesome! I will remember to look there next time :-) Thank you! I just updated it in the new commit.